### PR TITLE
Update the link to the demo in the README

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4868/
Test PASSed.

File: README.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ Juju-GUI bugs are tracked on `on Github Issues
 
 See also:
 
-- a `stable demo <http://jujucharms.com/>`_,
+- a `stable demo <http://demo.jujucharms.com/>`_,
```
Comment (hatched) [inline README.rst]: while you're here you might as well change http to https too :)

**Comment by anthonydillon** [comment]
@hatched There ya go

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4871/
Test PASSed.

**Comment by hatched** [comment]
Looks good! :D 

:shipit:  as trivial

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**